docs/content/items.md

```diff
@@ -118,6 +118,21 @@ URI fields link to an external website or online resource.
 
 You may add other fields by selecting a property from the list on the right. Browse fields by vocabulary (Dublin Core, Bibliographic Ontology, etc), or search in the *filter properties* bar above the list of properties and vocabularies.
 
+#### Value annotation
+When you input a value for a property associated with a resource, you may wish to include additional statements about that property. In the world of linked data, this process is know as [reification](https://www.w3.org/wiki/RdfReification). Each value can have any number of annotations.
```

Comment:
> When you input a value for a property associated with a resource, you may wish to include additional statements about that property

I understand where you're coming from, but it may be worth adding more detail:

> When you input a value for a property associated for a resource, you are making a _statement_ about that resource. If you choose, Omeka allows you to make statements about that statement. We call this value annotation. The advantage of value annotation is that you can choose to make ambiguous facts more concrete by annotating things like:
> - Provenance: Where is this fact from?
> - Time: When did this fact occur?
> - Location: What is the location associated with this fact?
> - Certainty: What is the confidence of this fact?
> - Type: What type of concept/thing is this fact?

> In the world of linked data, this process is know as [reification](https://www.w3.org/wiki/RdfReification). Each value can have any number of annotations.

Reply:
Makes sense. Will update.